lib/rototiller/task/params/env_var.rb

```diff
@@ -127,22 +107,22 @@ def reset
       end
 
       # @private
-      def check
+      def env_value_provided_by_user?
         # its possible that name could not be set
-        ENV.key?(@name) if @name
+        true if (ENV.key?(@name) if @name)
```

Comment:
what happens if ENV is not set and `!@name`?
maybe this always has a name, but i'd like to see this changed to `return true if...` followed by `return false`